### PR TITLE
Scaffold single exhibitor

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: lib/thumbnail-sizes.php [hunk 1]
```diff
@@ -9,11 +9,13 @@
   add_image_size( 'opengraph', 1200, 630, true );
   add_image_size( 'sponsor', 300, 100, false );
   add_image_size( 'splash', 1440, 550, true );
+
   add_image_size( 'col-3-crop', 252, 175, true );
   add_image_size( 'col-3-square', 252, 252, true );
   add_image_size( 'col-4-crop', 384, 230, true );
   add_image_size( 'col-6', 540, 9999, false );
   add_image_size( 'col-8', 770, 9999, false );
+  add_image_size( 'col-8-col-6', 770, 540, false );
```
Comment (varffs) [inline lib/thumbnail-sizes.php]: what does this mean?

File: partials/single-exhibitor/artists.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <div class="row">
+      <div class="col col-l col-l-12">
+        <h2><?php _e('[:en]Exhibiting Artists[:es]Artistas presentando'); ?></h2>
+      </div>
+    </div>
+    <div class="row">
+      <?php 
+        foreach ($artists as $artist) {
+      ?>
+      <div class="col col-l col-l-6 font-size-h3 margin-bottom-micro">
```
Comment (varffs) [inline partials/single-exhibitor/artists.php:18]: font size could go on the parent

File: partials/single-exhibitor/basic-info.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        <?php the_post_thumbnail('col-6'); ?>
+      </div>
+      <div class="col col-l col-l-6">
+        <div class="font-size-h4">
+          <?php the_content(); ?>
+        </div>
+
+        <?php 
+          if (!empty($website)) {
+            echo '<p class="font-size-h4"><a href="' . esc_url($website) . '">' . $website . '</a></p>';
```
Comment (varffs) [inline partials/single-exhibitor/basic-info.php]: target _blank?

File: partials/single-exhibitor/basic-info.php [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+          }
+
+          if (!empty($staff)) {
+        ?>
+        <ul>
+        <?php
+          foreach ($staff as $member) {
+        ?>
+          <li class="margin-bottom-micro">
+            <div class="font-size-h4"><?php echo $member['name']; ?></div>
```
Comment (varffs) [inline partials/single-exhibitor/basic-info.php]: why not just use a h4 for block level element of that size?

File: partials/single-exhibitor/basic-info.php (hunk 3 above)
Comment (mikeyrayvon) [inline partials/single-exhibitor/basic-info.php]: idk what do u think?

File: partials/single-exhibitor/basic-info.php (hunk 4 above)
Comment (mikeyrayvon) [inline partials/single-exhibitor/basic-info.php]: yea we could.  didnt know if it was worthy or not.

File: lib/thumbnail-sizes.php (hunk 1 above)
Comment (mikeyrayvon) [inline lib/thumbnail-sizes.php]: col-8 width, col-6 height. too weird?

File: partials/single-exhibitor/pagination.php [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      <div class="col col-l col-l-6 flex-row justify-end">
+<?php
+
+if ($current_year) {
+
+$args = array(
+  'post_type'       =>  'exhibitor',
+  'orderby'         =>  'title',
+  'order'           =>  'ASC',
+  'tax_query'       => array(
```
Comment (varffs) [inline partials/single-exhibitor/pagination.php]: I think this query needs `posts_per_page` -1 to work?

**Comment by varffs** [comment]
There are changes to the archive but it doesnt work for me, but the PR says single so I assume that is the scope of the PR.

**Comment by varffs** [comment]
One thing I notice is that as the year tax is set to be non hierarchical is has the tags formatting in WP which is kind of annoying as a UX as you have to manually enter it rather than checkbox which I think will feel more natural and be easier. I think just setting to hierarchical will fix that. Probably outside the scope of this PR but imo worth doing

File: lib/thumbnail-sizes.php (hunk 1 above)
Comment (varffs) [inline lib/thumbnail-sizes.php]: hmm. that also expresses as the ratio 4:3 which is kinda more common. I've been writing colx-4to3 as ratio sizes. A bit weird still but makes me think in common ratios when building which I think is good. A better syntax would be better tho

File: partials/single-exhibitor/basic-info.php (hunk 3 above)
Comment (varffs) [inline partials/single-exhibitor/basic-info.php]: Apparently cas doesnt like them but I think it is better, and tab clicking overrides it anyway which i imagine is how most people will want to experience the links.

File: partials/single-exhibitor/basic-info.php (hunk 4 above)
Comment (varffs) [inline partials/single-exhibitor/basic-info.php]: h4 is quite low down the semantic scale so would seem appropriate to use in this context

**Comment by mikeyrayvon** [comment]
ok i added a card to the trello about that

File: partials/single-exhibitor/basic-info.php (hunk 3 above)
Comment (mixcas) [inline partials/single-exhibitor/basic-info.php]: Is not that i don't like the behavior, its this:
[https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c#.ndhpbsrsi](https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c#.ndhpbsrsi)

Let's try using `rel="noopener noreferrer"`?

I mean, [Google doesn't care](https://sites.google.com/site/bughunteruniversity/nonvuln/phishing-with-window-opener) anyway.

**Pull request merged**